warehouse/integrations/bigquery/bigquery.go

```diff
@@ -115,23 +115,25 @@ func (bq *HandleT) DeleteTable(tableName string) (err error) {
 	return
 }
 
-func (bq *HandleT) CreateTable(tableName string, columnMap map[string]string) (err error) {
+func (bq *HandleT) CreateTable(tableName string, columnMap map[string]string) error {
```

Comment:
While writing the Unit test, we found that suppression is not happening properly.